apps/re_web/lib/exporters/trovit/plug.ex

```diff
@@ -16,25 +16,37 @@ defmodule ReWeb.Exporters.Trovit.Plug do
         _args
       ) do
     filters = %{cities_slug: [city_slug], states_slug: [state_slug]}
-
-    xml_listings =
-      filters
-      |> Exporter.exportable(query_params)
-      |> Trovit.export_listings_xml()
-
-    conn
-    |> put_resp_content_type("application/xml")
-    |> send_resp(200, xml_listings)
+    response_with_xml_listings(filters, query_params, conn)
   end
 
-  def call(conn, _args) do
+  def call(%Plug.Conn{path_info: [_h|_t], query_params: _} = conn, _args) do
```

Comment:
I'm not sure if this is the best approach to match any paths.

Reply:
```suggestion
  def call(conn, _args) do
```

Yes, give you one option, once we don't rely on these params we don't need to match them.